README.md

```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Amr coding with Jesus - Admission trials
+
+> Amr Coding with Jesus
+
+![screenshot](./app_screenshot.png)
+
+This project is a Microverse admission trial test
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Portfolio setup & mobile version skeleton!](https://amrhub.github.io/Coding-Amr-Jesus/)
```

Comment:
[Optional] I think you  forgot to change the Live Demo title would it be nicer if it was a descriptive title